tools/mapJHUCounties.js

```diff
@@ -26,17 +23,16 @@ async function findNameMapping() {
 
     if (point) {
       // Search within features
-      for (let feature of usCountyData.features) {
-        if (point && feature.geometry) {
-          let poly = turf.feature(feature.geometry);
+      for (const feature of usCountyData.features) {
+        if (feature.geometry) {
+          const poly = turf.feature(feature.geometry);
           if (turf.booleanPointInPolygon(point, poly)) {
             countyMap[location[STATE]] = feature.properties.name;
-            continue locationLoop;
+            continue;
```

Comment:
Erm, we want to continue the outer loop here; not the loop over `usCountyData`.

Reply:
Oh, gotcha. Yeah I never code in `continue` style, fixing now.